UKGovernmentBEIS/regulated-professions-register
# (1783) Add very basic functionality for adding a profession

## Changes in this PR

This builds the first screen for adding a new profession - "Top level details", as well as a Check your answers page and a Confirmation screen. There's still lots to do, as well as 3 or 4 more steps in the journey, but I'd like to get some feedback on how this is at the moment, and ideally get the PR merged before adding more, to keep things small and easy to review.

## Still to do (in future PRs)

- Switch "Nation" dropdown to a list of checkboxes, as there may be multiple Nations associated with a Profession.
- Put this journey behind an admin guard.
- **Question**: does it feel right to use the Industry ID value here, or should we look it up by name?

### Validations 
- Ensure session data is valid for the step the user is at in the journey
- Ensure we can't submit with an empty session
- Handle duplicate professions

## Screenshots

### Start the journey

![image](https://user-images.githubusercontent.com/19826940/145396470-48ee0f3f-22ac-4dcf-be52-8fb78a0ba1c9.png)

### Add top level information (Nations list will be replaced with a list of checkboxes in future)

![image](https://user-images.githubusercontent.com/19826940/145396923-5ffa8567-7a42-4164-91bd-6fe002815973.png)

### Check your answers

![image](https://user-images.githubusercontent.com/19826940/145396759-93bc60e7-3ea0-4d04-8458-98e2ed26bf83.png)

## Confirmation

![image](https://user-images.githubusercontent.com/19826940/145396825-81b09a51-aa04-4d41-9f89-0e30d9479928.png)
